alexthompson207/current-climate
# Feature/header component

# PR CurrentClimate

### What does this do?


### Are there any known bugs?


### Summary of the changes
Create a Header component folder with a JS and CSS file. Add basic styling and structure to header with title, overview, and logo/icon.

### What ticket(s) do the changes resolve?
closes #7 
Please Link Issues to this PR as necessary.

### Notes for the Reviewer
N/a

### How to Test Changes?
Open site locally by cloning or fetching branch, should see a basic header with title, overview and icon.

### Next Iterations
- Find font, color scheme for the website
- Connect app to get API information and display to the DOM
